packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -20,7 +20,7 @@
   "devDependencies": {
     "@types/chai": "^4.0.0",
     "@types/mocha": "^5",
-    "@types/node": "12.0.12",
+    "@types/node": "14.14.10",
```

Comment:
This change to do upstream dependency changes.

Reply:
The latest versions of VSCode still run on NodeJS 12.x.x so I would advice against changing this, you could accidentally start using new functionality that is not available.

Reply:
Fixed in '...revert...' commit